Admin/AdminInterface.php

```diff
@@ -266,18 +266,6 @@ public function getParentFieldDescription();
     public function hasParentFieldDescription();
 
     /**
-     * translate a message id.
-     *
-     * @param string $id
-     * @param array  $parameters
-     * @param null   $domain
-     * @param null   $locale
-     *
-     * @return string the translated string
-     */
-    public function trans($id, array $parameters = array(), $domain = null, $locale = null);
-
-    /**
```

Comment:
Interfaces are meant to be implemented. If I created a custom Admin class in my project, that doesn't inherit from `AbstractAdmin`, then this is a BC-break for me. But given the number of methods in this interface, I think nobody does that. Which leads me to the following question : what is the point of this interface?